guide/onboarding/funding-a-wallet.md

```diff
@@ -36,61 +36,19 @@ This section covers a basic mobile Lightning wallet funding flow that does not m
 
 {% include /tip/open.html label="⚡ Lightning services" icon="info" color="blue" %}
 
-Lightning services greatly simplify the UX of funding and using a Lightning wallet. We have highlighted the sections of funding a wallet with tip boxes like these that can benefit from using Lightning services. We have a dedicated page that covers funding a wallet using the outlined services.
+Lightning services greatly simplify the UX of funding and using a Lightning wallet. We have highlighted the sections of funding a wallet with tip boxes like these that can benefit from using Lightning services.
 
-{% include /tip/close.html %}
-
-<div class="image-slide-gallery">
-
-{% include picture.html
-   image = "/assets/images/guide/onboarding/funding-a-wallet/own-bitcoin.png"
-   retina = "/assets/images/guide/onboarding/funding-a-wallet/own-bitcoin@2x.png"
-   layout = "shadow"
-   caption = "Ask the user if they own bitcoin."
-   alt-text = "Screen asking the user if they own any bitcoin."
-   width = 250
-   height = 541
-%}
-
-{% include picture.html
-   image = "/assets/images/guide/onboarding/funding-a-wallet/funding-request.png"
-   retina = "/assets/images/guide/onboarding/funding-a-wallet/funding-request@2x.png"
-   layout = "shadow"
-   caption = "Show the user a payment request."
-   alt-text = "Screen showing payment request."
-   width = 250
-   height = 541
-%}
+We have a dedicated page that covers funding a wallet using the outlined services.
```

Comment:
We should add a link to this page here, I'm not sure which page is referred to.

Reply:
Ah, this page is LSP page, it's in #568, so I assume Bosch will do a follow up once that one is finished, to avoid broken links.